docs/pages/distribution/advanced-release-channels.md

```diff
@@ -79,7 +79,6 @@ Now we can see that our releases are available on the production channel.
 | publishedTime  | appVersion  | sdkVersion  | platform  | channel  | publicationId  |
 |---|---|---|---|---|---|---|
 | 2018-01-04T22:43:19.302Z  |  1.0.0 | 36.0.0 |  ios | production | d6b61741-a8dc-11e9-852a-3b0715b88238 |
-| 2018-01-04T22:43:46.714Z  |  1.0.0 | 36.0.0 |  android | production | 43d89652-a8dc-11e9-b565-4586c71668b9 |
```

Comment:
We keep this included to show that `expo publish:rollback --release-channel production --sdk-version 36.0.0` affects both platforms. We also state that we want to rollback for "all platforms," so if we want to change this to only include iOS in the table, then we should:

- edit this line- https://github.com/expo/expo/pull/9912/files#diff-babae4a4617e471f68965410d3253494R68
- explicitly state that `expo publish:rollback --release-channel production --sdk-version 36.0.0` will affect both android and ios unless one is specified with `--platform`

Reply:
I'm okay to keep the android line in the tabular output.  Removing the command that references it (the `expo publish:set ...` for the android push) is really the important thing to do.